ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -347,12 +355,12 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws
-- as: ocp-e2e-aws-ovn-arm64-techpreview-serial
+- as: ocp-e2e-aws-ovn-techpreview-serial-multi-a-a
```

Comment:
This one has "serial" at the end instead of after ocp-e2e and deployment type and payload are swapped according to naming covention.